lib/ansible/modules/meta.py

```diff
@@ -22,8 +22,11 @@
           points to implicitly trigger handler runs (after pre/post tasks, the final role execution, and the main tasks section of your plays).
         - V(refresh_inventory) (added in Ansible 2.0) forces the reload of the inventory, which in the case of dynamic inventory scripts means they will be
           re-executed. If the dynamic inventory script is using a cache, Ansible cannot know this and has no way of refreshing it (you can disable the cache
-          or, if available for your specific inventory datasource (e.g. aws), you can use the an inventory plugin instead of an inventory script).
-          This is mainly useful when additional hosts are created and users wish to use them instead of using the M(ansible.builtin.add_host) module.
+          or, if available for your specific inventory datasource (for example P(amazon.aws.aws_ec2#inventory)), you can use the an inventory plugin instead
+          of an inventory script). This is mainly useful when additional hosts are created and users wish to use them instead of using the
+          M(ansible.builtin.add_host) module.
+        - Note that neither V(refresh_inventory) nor the M(ansible.builtin.add_host) add hosts to the selected hosts for the current play.
```

Comment:
I'm not sure whether "selected hosts" is the right terminology for what is chosen from the `hosts` keyword of a play at the beginning of a play.

Reply:
```suggestion
        - Note that neither V(refresh_inventory) nor the M(ansible.builtin.add_host) add hosts to the hosts the current play iterates over.
```